modeling-cmds/src/def_enum.rs

```diff
@@ -1147,6 +1147,19 @@ define_modeling_cmd_enum! {
             Clone, Debug, Deserialize, JsonSchema, Serialize, ModelingCmdVariant,
         )]
         pub struct GetNumObjects;
+
+        /// Make a new path by offsetting an object by a given distance.
+        #[derive(Clone, Debug, Deserialize, JsonSchema, Serialize, ModelingCmdVariant)]
+        pub struct MakeOffsetPath {
+            /// The object that will be offset (can be a path, sketch, or the face of a solid)
+            pub object_id: Uuid,
+            /// If the object is a face, this is the ID of the face
+            pub face_id: Option<Uuid>,
+            /// The distance to offset the path (+ve for outset, -ve for inset)
+            pub offset: f64,
+            /// Id of the newly generated offset path
+            pub new_path_id: Uuid,
```

Comment:
Let's remove this, and use the ID of the command as the ID of the offset path (this is how StartPath and some other endpoints work). Pls add a note to the docstring on this struct saying that'll be the case.

Reply:
Oh so it isn't an argument of the endpoint? But when the backend calls the engine it inserts this to the engine call?

Reply:
The endpoints take two args:

1. The command ID (a UUID)
2. The command body (this struct)

Existing pattern (see MakePath, although we don't always stick to its example, but we're trying to) is that when the engine needs to create a new object, it should try to take the ID from the command.